content/v1.16/concepts/packages.md

```diff
@@ -54,6 +54,20 @@ spec:
   package: xpkg.upbound.io/upbound/platform-ref-aws:v0.6.0
 ```
 
+{{<hint "tip" >}}
```

Comment:
Could we add this section for v1.17 as well in `content/v1.17/concepts/packages.md`

Reply:
No, we did not cherry-pick the changes to previous releases

Reply:
Should we then maybe not have this addition in the v1.16 version content?

Reply:
thanks for the catch! fixed :)